### DISCUSSION TITLE
ERROR: Service 'odoo' failed to build : Build failed

### DISCUSSION THREAD

**Discussion by luismata01** [Tecnativa/doodba#457] [Q&A]
After Install Copier and Try to run
 docker-compose -f prod.yaml up -d
This message interrupt the process

ERROR: Service 'odoo' failed to build : Build failed

Odoo v14.0 CE

Can you Help me please

Luis

**Comment by joao-p-marques** [comment]
Please provide more info. and steps to reproduce

**Reply by luismata01** [reply]
It's a fresh install of Odoo v14 CE on a new VPS, and after install copier and the other reommendatios from Copier Repo I run this
docker-compose -f prod.yaml up -d
and in the moment that the instruction try to BUILD the image, this error appear.

**Reply by joao-p-marques** [reply]
We would still need more information to help, as the steps you mention work for our cases so far…

Please provide the full error logs of the build process
